Packs/Salesforce/Integrations/Salesforce_IAM/README.md

```diff
@@ -0,0 +1,223 @@
+> <i>Note:</i> This integration should be used along with our IAM premium pack. For further details, visit our IAM pack documentation.
+
+Integrate with Salesforce's services to perform Identity Lifecycle Management operations.
+For more information, please refer to the [Identity Lifecycle Management article](https://xsoar.pan.dev/docs/reference/articles/identity-lifecycle-management).
+
+## Configure Salesforce IAM on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Salesforce IAM.
+3. Click **Add instance** to create and configure a new integration instance.
+
+## Required Fields in Create User Command
+When creating a user in Salesforce there are mandatory fields that need to be set. Some of them are set with default values in the integration parameters:
+**Default Local Sid Key**, **Default Email Encoding Key** and **Default Language Locale Key**.
+**ProfileId** and **Timezone Sid Key** are also required, but are filled using the Salesforce mapper in the following manner:
+Duplicate the **DemoGenerateProfileId** and the **DemoGenerateTimeZone** automations, edit them according to your needs, and use them as transformers in the **User Profile - Salesforce (Outgoing)** mapper under the **ProfileId** and **TimeZoneSidKey** fields respectively.
```

Comment:
Lets change the name of the scripts to GeneratePollingId and GenerateTimeZone and suggest the user not to clone but detach the scripts and modify them. This way it will not require the user to update the mappers